### DISCUSSION TITLE
Explanation of all the default capture groups?

### DISCUSSION THREAD

**Discussion by ghost** [nvim-treesitter/nvim-treesitter#4275] [Q&A] [answered]
I've searched and searched but I can't find a good reference on what all the capture groups are supposed to mean. I can make some educated guesses about say `@type` or `@keyword`, but I'm completely lost on `@exception` and `@storageClass`. Is there a reference anywhere explaining all of them?

**Comment by clason** [comment, marked as answer]
https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#highlights is the best we have. Anything more detailed is impossible given that this list has to cover ~120 languages.

**Reply by ghost** [reply]
This is good, thanks.
